src/main/java/io/confluent/kafkarest/simpleconsumerspool/NaiveSimpleConsumerPool.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package io.confluent.kafkarest.simpleconsumerspool;
+
+import io.confluent.kafkarest.SimpleConsumerFactory;
+import io.confluent.kafkarest.SimpleFetcher;
+import kafka.cluster.Broker;
+import kafka.javaapi.consumer.SimpleConsumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * The NaiveSimpleConsumerPool instantiates a new SimpleConsumer each time it is called
+ */
+public class NaiveSimpleConsumerPool implements SimpleConsumerPool {
```

Comment:
I think that we can maybe get rid of this class and actually improve things. Here's why:

1. This naive approach is perhaps a bit too naive :) By not pooling at all, it means we never reuse the underlying resources. If we allow reuse then we avoid making a new network connection per request. I think anyone working with Kafka logs would prefer this given how much noise can be added to the log just from network connection events.
2. The SizeLimitedSimpleConsumerPool could give you similar, but better behavior and it's basically free. Just make an infinite size setting work (e.g. 0 indicates no limit, as it does given the current code in SimpleConsumerManager). This gives you unlimited size, but also reuses the connections as described above.
3. Once you do 2, then you don't need the SimpleConsumerPool interface since there's only one subclass. Instead, rename SizeLimitSimpleConsumerPool to be SimpleConsumerPool and we're left with only the one class.